Source/WebKit/Shared/WebMouseEvent.h

```diff
@@ -111,6 +113,7 @@ class WebMouseEvent : public WebEvent {
     WebCore::PointerID m_pointerId { WebCore::mousePointerID };
     String m_pointerType { WebCore::mousePointerEventType() };
     GestureWasCancelled m_gestureWasCancelled { GestureWasCancelled::No };
+    WebCore::IntPoint m_unadjustedMovementDelta;
```

Comment:
Move up next to m_deltaZ for better packing. I'd prefer this be an IntSize.

Reply:
Done in my latest push.